### ISSUE TITLE
changed tool icons doesn't always get reflected as expected

### ISSUE THREAD

**Issue by NicoledeGreef** [bcgov/smk-cli#82]
smk-cli v1.0.3/smk 1.0.7

created a new site with a single uploaded KML layer.  changed Identify tool icon.  new icon chosen doesn't show up in the site Preview:

<img width="540" alt="image" src="https://user-images.githubusercontent.com/26335003/148144062-9ccf5111-4dbd-47d3-a38d-552032ed4db2.png">

The "percent" icon is not showing up:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/26335003/148144183-d97c172d-78d3-46ba-9d0e-bd27dd6c13e3.png">

**Comment by NicoledeGreef** [comment]
another odd case i found when i randomly chose the "apps_outage" icon:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/26335003/148144412-8d6e0c89-b96e-4d4c-9eb1-d830ed5036ae.png">

two incorrect icons showed up:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/26335003/148144480-4f889e37-bfb1-4040-93b3-b744a20601c6.png">

**Comment by michaelpnelson** [comment]
We found that SMK uses an earlier version of the material icon font than SMK-CLI, and as a result, some icons (such as “percent”) are missing in the earlier version.

This was addressed by updating both SMK and SMK-CLI package dependency versions for all packages shared by both projects.

Related pull requests:

- https://github.com/bcgov/smk/pull/169
- https://github.com/bcgov/smk-cli/pull/87.
